SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -140,28 +140,22 @@ CDriver::CDriver(char* confFile, unsigned short val_nZone, SU2_Comm MPICommunica
 
     nInst[iZone] = config_container[iZone]->GetnTimeInstances();
 
-    geometry_container[iZone]    = new CGeometry**    [nInst[iZone]];
-    iteration_container[iZone]   = new CIteration*    [nInst[iZone]];
-    solver_container[iZone]      = new CSolver***     [nInst[iZone]];
-    integration_container[iZone] = new CIntegration** [nInst[iZone]];
-    numerics_container[iZone]    = new CNumerics****  [nInst[iZone]];
-    grid_movement[iZone]         = new CVolumetricMovement* [nInst[iZone]];
+    geometry_container[iZone]    = new CGeometry**    [nInst[iZone]] ();
+    iteration_container[iZone]   = new CIteration*    [nInst[iZone]] ();
+    solver_container[iZone]      = new CSolver***     [nInst[iZone]] ();
+    integration_container[iZone] = new CIntegration** [nInst[iZone]] ();
+    numerics_container[iZone]    = new CNumerics****  [nInst[iZone]] ();
```

Comment:
C++ understanding question: With `()` you call the default constructor, right? What is the benefit of that? Does it automatically initialize with `nullptr` as that set-ting was deleted below?

Reply:
Yes